core-bundle/contao/drivers/DC_Table.php

```diff
@@ -2388,13 +2393,14 @@ public function edit($intId=null, $ajaxId=null)
 </script>';
 		}
 
-		// Begin the form (-> DO NOT CHANGE THIS ORDER -> this way the onsubmit attribute of the form can be changed by a field)
+		// Begin the form (-> DO NOT CHANGE TH"IS ORDER -> this way the onsubmit attribute of the form can be changed by a field)
```

Comment:
```suggestion
		// Begin the form (-> DO NOT CHANGE THIS ORDER -> this way the onsubmit attribute of the form can be changed by a field)
```
Might have been a typo that you added